packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.component.test.js

```diff
@@ -55,16 +59,21 @@ describe('MultiSelectTag field', () => {
 		// eslint-disable-next-line react/no-render-return-value
 		const instance = ReactDOM.render(<MultiSelectTag {...props} />, node);
 		instance.updateSuggestions();
-		expect(instance.state.suggestions).toEqual([{ title: 'toto', value: 'titi' }]);
+		expect(instance.state.suggestions).toEqual([{ title: 'TITI', value: 'titi' }]);
 
 		// when : trigger a props update
 		ReactDOM.render(<MultiSelectTag {...props} value={['aze']} />, node);
 
 		// then
 		expect(instance.state.suggestions).toEqual([
-			{ title: 'toto', value: 'titi' },
-			{ title: 'tata', value: 'tutu' },
+			{ title: 'TITI', value: 'titi' },
+			{ title: 'TUTU', value: 'tutu' },
 		]);
+
+		// when : trigger a props update
+		ReactDOM.render(<MultiSelectTag {...props} value={['titi', 'tutu']} />, node);
```

Comment:
Don't change the existing tests, everything should work as before.

But add the tests that check your code
- filter with one of the current values --> no new
- filter with an exact suggestion patch —> no new
- filter without suggestion or value match --> new (already covered by an existing test)